src/Npgsql/NpgsqlLargeObjectStream.cs

```diff
@@ -9,26 +9,18 @@ namespace Npgsql
     /// An interface to remotely control the seekable stream for an opened large object on a PostgreSQL server.
     /// Note that the OpenRead/OpenReadWrite method as well as all operations performed on this stream must be wrapped inside a database transaction.
     /// </summary>
-    public sealed class NpgsqlLargeObjectStream : Stream
+    public sealed class NpgsqlLargeObjectStream : NpgsqlStream
     {
         readonly NpgsqlLargeObjectManager _manager;
         readonly int _fd;
         long _pos;
-        readonly bool _writeable;
-        bool _disposed;
 
         internal NpgsqlLargeObjectStream(NpgsqlLargeObjectManager manager, int fd, bool writeable)
+            : base(canWrite: writeable, canRead: true, canSeek: true)
```

Comment:
This makes code harder to maintain because you set up abilities here, but can easily forget about comments for properties below.

Reply:
My intent was to avoid losing existing comments. If this introduces more trouble than benefit, I prefer to remove them (and the useless properties override then).